components/devtools/lib.rs

```diff
@@ -552,3 +559,166 @@ fn handle_client(actors: Arc<Mutex<ActorRegistry>>, mut stream: TcpStream, strea
 
     actors.lock().unwrap().cleanup(stream_id);
 }
+
+#[derive(Debug, Default)]
+struct IdMap {
```

Comment:
Would you mind moving the new type and unit tests into a new file? I'm trying to keep this file from continuing to grow.

Reply:
done in a44813dcdd3b7f79832664a190c72cc6d7ef3f18